test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -265,6 +294,28 @@ export default function Main(props: WalletConnectionProps) {
         []
     );
 
+    // Refresh smartContractState periodically.
+    // eslint-disable-next-line consistent-return
+    useEffect(() => {
+        if (connection && credentialRegistryContratIndex) {
+            const interval = setInterval(async () => {
+                console.log('refreshing2');
+                if (credentialRegistryContratIndex !== 0) {
```

Comment:
Maybe we should use `undefined ` as the "not set" here, since in principle 0 is a valid index :)

Reply:
changed